src/Material_TEST.cc

```diff
@@ -289,3 +289,104 @@ TEST(DOMMaterial, InvalidSdf)
   sdf::Errors errors = material.Load(elem);
   EXPECT_EQ(sdf::ErrorCode::ELEMENT_INCORRECT_TYPE, errors[0].Code());
 }
+
+void PrintErrors(sdf::Errors &_errors)
+{
+  for (const auto &e : _errors)
+    std::cout << e << std::endl;
+}
+
+/////////////////////////////////////////////////
+TEST(DOMMaterial, Colors)
+{
+  sdf::ElementPtr elem(new sdf::Element());
+  elem->SetName("material");
+
+  // invalid diffuse (empty)
+  sdf::ElementPtr elemDiffuse(new sdf::Element());
+  elemDiffuse->SetName("diffuse");
+  elemDiffuse->AddValue("string", "", false, "description");
+  elem->InsertElement(elemDiffuse);
+
+  sdf::Material material;
+  sdf::Errors errors = material.Load(elem);
+  PrintErrors(errors);
```

Comment:
tip: You should be able to do `std::cout << errors << std::endl;`

Reply:
Thanks!